src/components/Composer.vue

```diff
@@ -6,12 +6,12 @@
 				{{ t('mail', 'From') }}
 			</label>
 			<div class="composer-fields--custom">
-				<Select
+				<NcSelect
```

Comment:
![image](https://github.com/nextcloud/mail/assets/76647797/90c5db79-181d-4e9e-8e76-c3d0c2658cae)

When clicking on the empty line underneath, an empty select rectangle opens up, don't think this is supposed to happen

Reply:
Yes, composer is still broken